tests/datasets_validation/test_enriched_datasets.py

```diff
@@ -0,0 +1,79 @@
+import great_expectations as ge
+from sem_covid.services.store_registry import store_registry
+
+es_store = store_registry.es_index_store()
+ENRICHMENT_COLUMNS = ['businesses', 'citizens', 'workers',
+                      'category', 'subcategory', 'type_of_measure', 'funding']
+COMMON_COLUMNS = ['title']
+DATA_FRAME_INDEX_NAME = ['ds_eu_cellar_enriched', 'ds_eu_timeline_enriched', 'ds_ireland_timeline_enriched']
+EU_CELLAR_COLUMNS = ['eurovoc_concepts', 'title', 'content', 'subject_matter_labels']
+EU_TIMELINE_COLUMNS = ['title', 'detail_content', 'topics', 'abstract']
+IRELAND_TIMELINE_COLUMNS = ['title', 'content', 'page_type', 'keyword']
+
+
+def validate_enriched_dataset(data_frame_index_name):
+    df_from_elastic = es_store.get_dataframe(index_name=data_frame_index_name)
+    gdf = ge.from_pandas(df_from_elastic)
+    # they contain the enrichment columns
+    assert gdf.expect_table_columns_to_match_set(column_set=ENRICHMENT_COLUMNS,
+                                                 exact_match=False).success
+    # no more than 1% of titles shall be missing
+    assert gdf.expect_column_values_to_not_be_null(column="title", mostly=0.99).success
+    # no more than 1% of content shall be missing
+    assert gdf.expect_column_values_to_not_be_null(column="content", mostly=0.99).success
```

Comment:
can we test if the title is the empty string ""?
this is technically not a null value but semantically empty.